lib/tasks/payment_breakdown.rake

```diff
@@ -44,6 +49,10 @@ namespace :payment_calculation do
       ]
     end
 
+    uplift_payment = breakdown[:uplift].each_with_object({}) do |(k, v), h|
```

Comment:
There's a fair amount of code in this rake task, if we change it do any tests break?

Reply:
No. Tests are using the same calls as the rake task, but not the rake task itself.